src/tui/table.rs

```diff
@@ -0,0 +1,53 @@
+use std::fmt::Display;
+
+use comfy_table::{presets::ASCII_MARKDOWN, Cell, ContentArrangement, Table};
+
+pub struct RusticTable {
+    table: Table,
+}
+
+impl RusticTable {
+    pub fn new() -> Self {
+        let mut table = Table::new();
+        table
+            .load_preset(ASCII_MARKDOWN)
+            .set_content_arrangement(ContentArrangement::Dynamic);
+
+        Self { table }
+    }
+
+    pub fn set_titles<I, D>(&mut self, titles: I)
```

Comment:
I would use the existing `set_header` from Table.
Moreover, this hard-encodes the format of the title - bold might be fine, but left/right-aligned surely doesn't fit for all tables. Better use the provided format methods at the place where the table is defined (i.e. in `repoinfo.rs`).